flit_core/flit_core/common.py

```diff
@@ -172,6 +157,20 @@ def get_docstring_and_version_via_ast(target):
     return ast.get_docstring(node), version
 
 
+def is_version_str_node(node, /):
```

Comment:
```suggestion
def is_version_str_node(node):
```

For compatibility with older Python versions.

We're probably coming to the point where even for flit_core I'll drop support for some older Python versions, but let's work that out separately. I generally find it easier to support a pretty broad range of Python versions than to deal with people getting stuck on an old version of a package I maintain because I dropped support for older Python.

Reply:
Done

Reply:
The convention for mypy is that a name starting with `__*` is positional-only. So as far as mypy is concerned, `...(__node):` is identical to `...(node, /)`. (If you really wanted it to be positional only).

Reply:
Indeed, but it doesn't matter that much (all of this is internal-only, anyway).